### ISSUE TITLE
Expected time taken for TOBIAS to run

### ISSUE THREAD

**Issue by loganminhdang** [loosolab/TOBIAS#155]
Thank you for creating this series of tools! I'm writing to enquire about the computational resource required by the TOBIAS pipeline. I have been trying to run the first few steps (ATACorrect, Scorebigwig) and realized that it took me more than an hour (with no sign of completion) to run ATACorrect. I was able to analyse your test data in less than a minute, so this puzzles me. Granted, my dataset is large compared to yours, but as I'm using an HPC cluster with hundreds of gigabytes in memory, so I reckon this may not be a memory issue but rather, an issue with my data. 

When I run any step of the pipeline, I'm usually stuck (for one hour plus with no signs of analysis completion) after seeing this message: 

2022-07-10 22:19:59 (237479) [INFO]     ----- Processing input data -----
2022-07-10 22:19:59 (237479) [INFO]     Reading info from .bam file
2022-07-10 22:19:59 (237479) [INFO]     Reading info from .fasta file
2022-07-10 22:19:59 (237479) [INFO]     Processing input/output regions

Can you please advise?

**Comment by quangdang1998** [comment]
To provide more context: 
- I was able to analyse the test data just fine, so it's probably not an issue with the software installation and functioning. 
- I used Genrich to perform the peak-calling, which should already correct for the Tn5 insertion bias. Maybe, this is where the issue lies. I could reperform peak-calling using MACS2. 
- I used bam files which were merged for replicates and removed duplicated regions using Picard.
